### ISSUE TITLE
App for Android

### ISSUE THREAD

**Issue by Lonerider2010** [cooklang/cooklang-ios-app#18]
Are there plans to create a similar app in Android?

**Comment by dubadub** [comment]
Yes, we'd been working on it and planned to release the first version soon. The sad truth is that our team member who works on it lives in Ukraine and of course now there're other priorities...

**Comment by Lonerider2010** [comment]
Ok, understand this and yes, that is more than sad.
Just to mention it: if you get to the state of releasing the app, please publish it on [F-Droid](https://f-droid.org/), an alternative catalog of FOSS applications for Android.

**Comment by ptrkaz** [comment]
Hey @dubadub, can I be of any help in this matter? I have android experience and was thinking bout building an iOS clone, but if there is already a WIP project, I might as well help you guys out.

**Comment by dubadub** [comment]
Hi @ptrkaz, I replied to you on Discord, could you please check?

**Comment by dubadub** [comment]
App for Android's published now https://play.google.com/store/apps/details?id=md.cook.android
